1-getting-started-lessons/1-intro-to-programming-languages/translations/README.nl.md

```diff
@@ -0,0 +1,193 @@
+
+### Populaire Command Lines
+
+Beschikbaarheid van command lines verschilt op basis van het besturingssysteem wat je gebruikt.
+
+*💻 = is geïnstalleerd met het besturingssysteem.*
+
+#### Windows
+
+- [Powershell](https://docs.microsoft.com/powershell/scripting/overview?view=powershell-7) 💻
+- [Command Line](https://docs.microsoft.com/windows-server/administration/windows-commands/windows-commands) (ook bekend als CMD) 💻
+- [Windows Terminal](https://docs.microsoft.com/windows/terminal/)
+- [mintty](https://mintty.github.io/)
+  
+#### MacOS
+
+- [Terminal](https://support.apple.com/guide/terminal/open-or-quit-terminal-apd5265185d-f365-44cb-8b09-71a064a42125/mac) 💻
+- [iTerm](https://iterm2.com/)
+- [Powershell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-macos?view=powershell-7)
+
+#### Linux
+
+- [Bash](https://www.gnu.org/software/bash/manual/html_node/index.html) 💻
+- [KDE Konsole](https://docs.kde.org/trunk5/en/applications/konsole/index.html)
+- [Powershell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-7)
+
+#### Populaire Command Line Applicaties
+
+- [Git](https://git-scm.com/) (💻 op de meeste besturingssystemen)
+- [NPM](https://www.npmjs.com/)
+- [Yarn](https://classic.yarnpkg.com/en/docs/cli/)
+
+### Documentatie
+
+Wanneer een ontwikkelaar iets nieuws wil leren dan zullen ze waarschijnlijk de bijbehorende documentatie raadplegen. Ontwikkelaars zijn erg afhankelijk van goede documentatie, wat hun verder helpt om nieuwe programmeertalen of applicaties te gebruiken waardoor ze een beter inzicht krijgen hoe dit toe te passen in hun dagelijks werk.
+
+#### Populaire Documentatie over Web Ontwikkeling
+
+- [Mozilla Developer Network](https://developer.mozilla.org/docs/Web)
+- [Frontend Masters](https://frontendmasters.com/learn/)
+
+✅ Doe wat onderzoek: Nu je de basis weet van de omgeving van een web ontwikkelaar, probeer eens uit te zoeken in hoeverre dit verschilt van een omgeving voor een web designer.
+
+---
+
+## 🚀 Uitdaging
+
+Vergelijk wat programmeertalen. Wat zijn unieke kenmerken van JavaScript vs. Java. En het verschil tussen COBOL en Go?
+
+## Post-Lecture Quiz
+[Post-lecture quiz](../.github/post-lecture-quiz.md)
+
+## Review & Zelfstudie
+
+Bestudeer wat voor verschillende programmeertalen er zijn. Kies 3 talen en probeer een regel code te schrijven in elke taal. Wat heb je geleerd?
```

Comment:
Ik zal deze suggestie ook in het origineel doen, maar: stel voor een "Hello World" tutorial op te zoeken voor de 3 talen, zo krijg je een goed idee wat er voor nodig is in elk van de 3 uitgekozen talen om iets in the browser te printen.